application/controllers/admin/Translate.php

```diff
@@ -931,14 +934,14 @@ private function displayTranslateFields(
             $translateoutput .= "<td class='col-sm-2'>" . htmlspecialchars($rowfrom['answer']) . " (" . $rowfrom['qid'] . ") </td>";
         }
         if ($type == 'question_help' || $type == 'question') {
-            $translateoutput .= "<td class='col-sm-2'>" . htmlspecialchars($rowfrom['question']) . " ({$rowfrom['qid']}) </td>";
+            $translateoutput .= "<td class='col-sm-2'>" . htmlspecialchars($rowfrom['title']) . " ({$rowfrom['qid']}) </td>";
```

Comment:
This part replace question text by quetion title ?

related to the issue fixed ?